src/features/classification/applied-by-ai-classification-reason/AppliedByAiClassificationReason.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { AnswerContent, References } from '@box/box-ai-content-answers';
+import { Card, Text } from '@box/blueprint-web';
+import BoxAIIconColor from '@box/blueprint-web-assets/icons/Logo/BoxAiLogo';
+import { Size5 } from '@box/blueprint-web-assets/tokens/tokens';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+
+import { isValidDate } from '../../../utils/datetime';
+import type { AiClassificationReason } from '../types';
+
+import messages from './messages';
+
+import './AppliedByAiClassificationReason.scss';
+
+const AppliedByAiClassificationReason = ({ answer, modifiedAt, citations }: AiClassificationReason) => {
```

Comment:
nit: I think typically it's preferred to keep a separate props declaration even if it matches an existing type. So could maybe do something like:

```
export type AppliedByAiClassificationReasonProps = AiClassificationReason;
```